<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Registration</class>
    <widget class="QDialog" name="Registration">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>370</width>
                <height>267</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Registration</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget_2">
            <property name="geometry">
                <rect>
                    <x>40</x>
                    <y>20</y>
                    <width>291</width>
                    <height>190</height>
                </rect>
            </property>
            <layout class="QVBoxLayout" name="authorizationLayout">
                <property name="spacing">
                    <number>7</number>
                </property>
                <property name="sizeConstraint">
                    <enum>QLayout::SetMaximumSize</enum>
                </property>
                <item>
                    <layout class="QVBoxLayout" name="verticalLayout_3">
                        <property name="sizeConstraint">
                            <enum>QLayout::SetDefaultConstraint</enum>
                        </property>
                        <item>
                            <widget class="QLabel" name="label_3">
                                <property name="cursor">
                                    <cursorShape>ArrowCursor</cursorShape>
                                </property>
                                <property name="mouseTracking">
                                    <bool>false</bool>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Plain</enum>
                                </property>
                                <property name="text">
                                    <string>Придумайте логин</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>false</bool>
                                </property>
                                <property name="wordWrap">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="login">
                                <property name="cursor">
                                    <cursorShape>ArrowCursor</cursorShape>
                                </property>
                                <property name="mouseTracking">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label">
                                <property name="cursor">
                                    <cursorShape>ArrowCursor</cursorShape>
                                </property>
                                <property name="mouseTracking">
                                    <bool>false</bool>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Plain</enum>
                                </property>
                                <property name="text">
                                    <string>Введите Ваше имя</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>false</bool>
                                </property>
                                <property name="wordWrap">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="input_name">
                                <property name="cursor">
                                    <cursorShape>ArrowCursor</cursorShape>
                                </property>
                                <property name="mouseTracking">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLabel" name="label_2">
                                <property name="cursor">
                                    <cursorShape>ArrowCursor</cursorShape>
                                </property>
                                <property name="mouseTracking">
                                    <bool>false</bool>
                                </property>
                                <property name="frameShadow">
                                    <enum>QFrame::Plain</enum>
                                </property>
                                <property name="text">
                                    <string>Введите Вашу фамилию</string>
                                </property>
                                <property name="scaledContents">
                                    <bool>false</bool>
                                </property>
                                <property name="wordWrap">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <widget class="QLineEdit" name="input_surname">
                                <property name="cursor">
                                    <cursorShape>ArrowCursor</cursorShape>
                                </property>
                                <property name="mouseTracking">
                                    <bool>false</bool>
                                </property>
                            </widget>
                        </item>
                        <item>
                            <layout class="QHBoxLayout" name="horizontalLayout">
                                <item>
                                    <widget class="QPushButton" name="sign_up">
                                        <property name="text">
                                            <string>Создать пользователя</string>
                                        </property>
                                    </widget>
                                </item>
                            </layout>
                        </item>
                    </layout>
                </item>
            </layout>
        </widget>
        <widget class="QPushButton" name="return_back">
            <property name="geometry">
                <rect>
                    <x>110</x>
                    <y>230</y>
                    <width>135</width>
                    <height>28</height>
                </rect>
            </property>
            <property name="text">
                <string>Вернуться</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
